nms/app/packages/magmalte/app/views/events/EventsTable.js

```diff
@@ -228,6 +297,17 @@ export default function EventsTable(props: EventTableProps) {
       ? streamNameSessiond
       : streamNameMagmad + ',' + streamNameSessiond;
 
+  const {
+    startDate,
+    endDate,
+    setStartDate,
+    setEndDate,
+    isAutorefreshing,
```

Comment:
Wouldn't it be better for isAutoRefreshing to be passed in as a prop here ? and the component (checkbox) owning the state for isAutoRefreshing ? Curious on why this is being returned

Reply:
@karthiksubraveti, do you mean that the hook should be used by the new checkbox component?

Reply:
We could have the component enclosing the checkbox to own the state for autorefresh and we can pass in the autorefresh value into the hook.